test/commands/extended/replay_bundle_test.py

```diff
@@ -593,10 +593,20 @@ def test_happy_path(self):
       })
 
     send_trytes_response = {
-      'trytes': bundle.as_tryte_strings(head_to_tail=True),
+      'trytes': bundle.as_tryte_strings(),
     }
 
-    mock_send_trytes = mock.Mock(return_value=send_trytes_response)
+    def mock_send_trytes(self,request):
+      """
+      Ensures that the correct trytes are sent to the ``sendTrytes`` command
+      to replay the bundle.
+
+      References:
+        - https://github.com/iotaledger/iota.lib.py/issues/74
+      """
+      from unittest import TestCase
+      TestCase().assertEqual(request['trytes'], send_trytes_response['trytes'])
```

Comment:
Python 2.7 doesn't like that so much.  Maybe something like this:

```python
def mock_send_trytes(_, request):
  ...
  self.assertEqual(...)
```

Reply:
Aye.  I admit; it feels like a hack, either way, but that's why I'm so keen on getting #43 set up (: